### DISCUSSION TITLE
Problem from noob - changing background color

### DISCUSSION THREAD

**Discussion by hanny224** [community/community#21762] [New to GitHub]
<p>Hey guys, i’ve been trying to change the background color of my github html file for 2 days now but I just can’t figure it out. </p>
<p>&lt;html&gt;<br>
&lt;body&gt;<br>
&lt;body bgcolor = “LightSkyBlue”&gt;<br>
&lt;h1 align=“center”&gt;My name is Hanny 224. Welcome to my site!&lt;/h1&gt;<br>
&lt;p align=“center”&gt;<br>
&lt;img width=“460” height=“300” src=“Hehexd”&gt;<br>
&lt;/p&gt;<br>
&lt;br&gt;</p>
<p>&lt;h2&gt;&lt;b&gt;About Me&lt;/b&gt;&lt;/h2&gt;<br>
&lt;p&gt; I like to swim and listen to music. My favorite food is pizza.&lt;/p&gt;<br>
&lt;h2&gt;My favorite website&lt;/h2&gt;<br>
&lt;a href=“<a href="https://www.romphim.com/" rel="nofollow noopener">https://www.romphim.com/</a>”&gt;Click here to go to my favorite site!&lt;/a&gt;<br>
&lt;/body&gt;<br>
&lt;/html&gt;</p>
<p>My code looks along the lines of this. It seems like it worked for most other people after using the body bgcolor, but i have no idea why it isn’t working for me. Can anyone help me out? Thank you so much</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/hanny224">@hanny224</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>Did you tried using the hexadecimal value?</p>
<p>Change the value of bgcolor with this <span class="hashtag">#2432BE</span></p>
<pre><code>&lt;html&gt;
&lt;body&gt;
&lt;body bgcolor = **#2432BE** &gt;
&lt;h1 align="center"&gt;My name is Hanny 224. Welcome to my site!&lt;/h1&gt;
&lt;p align="center"&gt;
&lt;img width="460" height="300" src="Hehexd"&gt;
&lt;/p&gt;



&lt;h2&gt;&lt;b&gt;About Me&lt;/b&gt;&lt;/h2&gt;
&lt;p&gt; I like to swim and listen to music. My favorite food is pizza.&lt;/p&gt;
&lt;h2&gt;My favorite website&lt;/h2&gt;
&lt;a href="https://www.romphim.com/"&gt;Click here to go to my favorite site!&lt;/a&gt;
&lt;/body&gt;
&lt;/html&gt;
</code></pre>
<p>If you want a different type of blue you can use this tool <a href="https://mycolor.space/" rel="nofollow noopener">mycolor.space</a>, and find the hexadecimal value of the color you want!</p>
<p>Hope to solve your problem :slight_smile:</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>
